src/libraries/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -336,6 +364,12 @@ public static partial class Vector
         public static System.Numerics.Vector<System.UInt32> Narrow(System.Numerics.Vector<System.UInt64> low, System.Numerics.Vector<System.UInt64> high) { throw null; }
         public static System.Numerics.Vector<T> Negate<T>(System.Numerics.Vector<T> value) { throw null; }
         public static System.Numerics.Vector<T> OnesComplement<T>(System.Numerics.Vector<T> value) { throw null; }
+        public static System.Numerics.Vector<double> RadiansToDegrees(System.Numerics.Vector<double> radians) { throw null; }
+        public static System.Numerics.Vector<float> RadiansToDegrees(System.Numerics.Vector<float> radians) { throw null; }
+        public static System.Numerics.Vector<double> Round(System.Numerics.Vector<double> vector) { throw null; }
+        public static System.Numerics.Vector<float> Round(System.Numerics.Vector<float> vector) { throw null; }
+        public static System.Numerics.Vector<double> Round(System.Numerics.Vector<double> vector, System.MidpointRounding mode) { throw null; }
+        public static System.Numerics.Vector<float> Round(System.Numerics.Vector<float> vector, System.MidpointRounding mode) { throw null; }
```

Comment:
The overloads that take a number of digits don't make sense here?

Reply:
They do and are approved, but I didn't cover everything in this PR just the ones that were trivial and would have the biggest impact.

The more complex APIs like the trigonometric, hyperbolic, and overloads like `Round(vector, int digits, MidpointRounding mode)` will come in a follow up PR.